src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -93,7 +94,10 @@ const TickDisplay = (() => {
                 if (!CommonFunctions.isVisible(el_chart_container)) return;
                 clearTimeout(resize_timeout);
                 resize_timeout = setTimeout(() => {
-                    const { offsetWidth, offsetHeight } = el_chart_container;
+                    const { offsetWidth, offsetHeight } = el_chart_container || {};
+                    if (isEmptyObject(chart) || !offsetWidth || !offsetHeight) {
+                        return;
+                    }
                     chart.setSize(offsetWidth, offsetHeight);
```

Comment:
Seems, that it's a place where we pass `{}` / `null` / `undefined` into the `hightstock` library and it might cause the cascade of errors. I added a check for such values, but recommend to update the version of the library to the latest and/or put it into a whitelist in trackJS as this place is not the only one, which cause this error.
![Screenshot 2024-06-03 at 10 24 14 AM](https://github.com/deriv-com/smarttrader/assets/121025168/bc6b1e1a-ce3d-4d3f-b437-b85067ae3833)
<img width="911" alt="Screenshot 2024-06-03 at 10 26 56 AM" src="https://github.com/deriv-com/smarttrader/assets/121025168/f663d794-a78f-443d-a1b6-867b9295b57f">